website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -139,3 +144,11 @@ Once all the steps above have been completed, it's time to signal to the communi
 
 - [@reactnative](https://twitter.com/reactnative) on twitter
 - RN Discord `#releases-coordination`
+
+### 10. Bump minor versions of all monorepo packages
+
+The packages in the `react-native` monorepo should be always one minor version ahead of the latest or RC version. Once you're done with releasing the initial RC0, you should:
+
+- Create a new branch in `react-native` from `main` in your own fork.
+- Run `npm run bump-all-updated-packages --release-branch-cutoff` to bump the minor versions of all packages.
```

Comment:
```suggestion
- Run `npm run bump-all-updated-packages -- --release-branch-cutoff` to bump the minor versions of all packages.
```

For npm we need add explicit `--`, because `node` accepts arguments in a form like `--<arg>`